app/src/main/assets/ime/text/characters/jcuken_russian.json

```
@@ -0,0 +1,46 @@
+{
+  "type": "characters",
+  "name": "jcuken_russian",
+  "authors": [ "williamtheaker" ],
+  "direction": "ltr",
+  "arrangement": [
+    [
+      { "code":  1081, "label": "й" },
+      { "code":  1094, "label": "ц" },
+      { "code":  1091, "label": "у" },
+      { "code":  1082, "label": "к" },
+      { "code":  1077, "label": "е" },
+      { "code":  1085, "label": "н" },
+      { "code":  1075, "label": "г" },
+      { "code":  1096, "label": "ш" },
+      { "code":  1097, "label": "щ" },
+      { "code":  1079, "label": "з" },
+      { "code":  1093, "label": "х" },
+      { "code":  1066, "label": "Ъ" }
```

here you use Ъ (capital hard sign) when it should be ъ (small soft sign); this looks odd on the keyboard and is definitely a mistake which should be fixed before merging.
```suggestion
      { "code":  1066, "label": "ъ" }
```

---

Good catch thank you!